File: 공예진/programmers/P181884.java
//문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181884
//시간: 0.03ms
//메모리: 80.7MB

class Solution {
    public int solution(int[] numbers, int n) {
        int answer = 0;
        for (int i = 0; i < numbers.length; i++) {
            answer += numbers[i];
            if (answer > n) {
                break;
            }
        }
        return answer;
    }
}

File: 강재연/programmers/P181884.js
// 문제 링크: https://school.programmers.co.kr/learn/courses/30/lessons/181884
// 시간: 0.04 ms
// 메모리: 34.4 MB

function solution(numbers, n) {
    let answer = 0;
    numbers.forEach((num)=>{
        if(answer <= n) {
            answer += num;
        }
    });
    
    return answer;
}
